newIDE/app/src/ProjectsStorage/StorageProviderListItem.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+import AuthenticatedUserContext from '../Profile/AuthenticatedUserContext';
+import { Line, Spacer } from '../UI/Grid';
+import Text from '../UI/Text';
+import RaisedButton from '../UI/RaisedButton';
+import FlatButton from '../UI/FlatButton';
+
+import { type StorageProvider } from '.';
+import { makeStyles } from '@material-ui/styles';
+
+type Props = {|
+  storageProvider: StorageProvider,
+  onChooseProvider: (storageProvider: StorageProvider) => void,
+|};
+
+const useListItemStyles = makeStyles(theme => {
+  return {
+    root: {
+      border: `1px solid ${theme.palette.divider}`,
+      borderRadius: '5px',
```

Comment:
This is not a UI component 🧐
I'll give this a pass but let's think about it should this appear in another dialog in the future.

Reply:
I thought about creating a UI component but it felt very specialized for this use.
We could create a UI component if we want to use it elsewhere. At that time, it will be easier to know what needs to be put in common

Reply:
Ok!